src/plugin/plugin_pose_MARS.py

```diff
@@ -92,6 +104,101 @@ def loadPoses(self, parent_widget: QWidget, path: str):
                 frame_polys.append(poly)
             self.pose_polys.append(frame_polys)
 
+    def exportPosesToNWBFile(self, id: int, nwbFile: NWBFile):
+
+        processing_module_name = f"Pose data for video {os.path.basename(self.video_path)}"
+        for mouse_ix in range(self.num_mice):
+            pose_estimation_series = []
+            pose_estimation_series.append(
+                PoseEstimationSeries(
+                    name = 'nose',
+                    description = 'Pose keypoint placed aroud nose',
+                    data = self.keypoints[:,mouse_ix,:,0],
+                    reference_frame = '(0,0,0) corresponds to ...',
+                    timestamps = np.arange(self.num_frames, dtype=float), 
+                    confidence = self.confidence[:,mouse_ix,0]
+                )
+            )
+            pose_estimation_series.append(
+                PoseEstimationSeries(
+                    name = 'left ear',
+                    description = 'Pose keypoint placed aroud left ear',
+                    data = self.keypoints[:,mouse_ix,:,1],
+                    reference_frame = '(0,0,0) corresponds to ...',
+                    timestamps = np.arange(self.num_frames, dtype=float),
+                    confidence = self.confidence[:,mouse_ix,1]
+                )
+            )
+            pose_estimation_series.append(
+                PoseEstimationSeries(
+                    name = 'right ear',
+                    description = 'Pose keypoint placed aroud right ear',
+                    data = self.keypoints[:,mouse_ix,:,2],
+                    reference_frame = '(0,0,0) corresponds to ...',
+                    timestamps = np.arange(self.num_frames, dtype=float),
+                    confidence = self.confidence[:,mouse_ix,2]
+                )
+            )
+            pose_estimation_series.append(
+                PoseEstimationSeries(
+                    name = 'neck',
+                    description = 'Pose keypoint placed aroud neck',
+                    data = self.keypoints[:,mouse_ix,:,3],
+                    reference_frame = '(0,0,0) corresponds to ...',
+                    timestamps = np.arange(self.num_frames, dtype=float),
+                    confidence = self.confidence[:,mouse_ix,3]
+                )
+            )
+            pose_estimation_series.append(
+                PoseEstimationSeries(
+                    name = 'left hip',
+                    description = 'Pose keypoint placed aroud left hip',
```

Comment:
check description everywhere (aroud to around)
reference_frame to top left corner